R/sjTabItemAnalysis.R

```diff
@@ -92,24 +92,22 @@
 #'             }
 #' 
 #' @examples
-#' # Data from the EUROFAMCARE sample dataset
-#' library(sjmisc)
-#' data(efc)
-#' 
-#' # retrieve variable and value labels
+#' Data from the EUROFAMCARE sample dataset
+#' library(sjmisc)data(efc)
+#' retrieve variable and value labels
 #' varlabs <- get_label(efc)
 #' 
 #' # recveive first item of COPE-index scale
 #' start <- which(colnames(efc) == "c82cop1")
 #' # recveive last item of COPE-index scale
 #' end <- which(colnames(efc) == "c90cop9")
-#'  
+#' 
 #' # create data frame with COPE-index scale
 #' mydf <- data.frame(efc[, c(start:end)])
 #' colnames(mydf) <- varlabs[c(start:end)]
-#' 
-#' \dontrun{
-#' sjt.itemanalysis(mydf)
+#' #' #'
+#' #' #' \dontrun{
```

Comment:
I think my answer was a bit confusing. The dontrun-tag has no consequences, when copy&paste examples from docs to console. It's just that example sourrounded by dontrun are not run inside CRAN checks (R CMD check). You don't need to comment/uncomment to test these examples (unless you want to test using R CMD check).